src/test/java/StudentUnitTests.java

```diff
@@ -0,0 +1,46 @@
+import logic.CoursesStatus;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import studentInfo.Student;
+import taskItems.Course;
+import taskItems.TrainingCenterTimetable;
+
+import java.time.LocalDateTime;
+
+import static logic.CoursesStatus.getDiffWorkTimeInHours;
+import static logic.CoursesStatus.timeProgramPassed;
+
+public class StudentUnitTests {
+
+    private CoursesStatus coursesStatus;
+    private LocalDateTime todayTime = LocalDateTime.now();
+    private LocalDateTime todayPlusTenDaysTime = LocalDateTime.now().plusDays(10);
+    private static final int expectedWorkingHoursDiff = 32;
```

Comment:
please follow Java code convention for the constant names

Reply:
done